field_rrule/demo/res_partner.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
this unfortunately doesn't work, because modules are loaded server wide, but demo or not is a property of a database. no chance to determine which database is going to be used during server startup